src/main/menus/edit.js

```diff
@@ -88,5 +88,20 @@ export default {
         actions.insertImage(browserWindow, 'upload')
       }
     }]
+  }, {
+    type: 'separator'
+  }, {
```

Comment:
A little confused.

Since we have already set line break in the user preference file, why do we need to set it here?

Reply:
The user settings option is used for new files and the fallback line ending. The menu entry is to change the current file line ending - I thought it's a nice feature and facilitate testing.

Reply:
You mean that the line break menu is only control the current opened file?

and if you change line break in menu, and it will not change the line break preference in preference file?

Reply:
That's right, the menu is only used for the current file and independent from settings (it doesn't change settings).

Reply:
@fxha OK, I'll review these codes which you modified this evening.